jackal_control/launch/teleop.launch

```diff
@@ -4,8 +4,9 @@
 
   <group ns="bluetooth_teleop" if="$(arg joystick)">
 
-    <group unless="$(optenv JACKAL_PS3 0)" >
+    <group unless="$(optenv JACKAL_PS4 0)" >
       <rosparam command="load" file="$(find jackal_control)/config/teleop_ps4.yaml" />
```

Comment:
Unrelated to this change, but for paths within the package, it may now be preferable to use `$(dirname)`. This line would become:

    <rosparam command="load" file="$(dirname)/../config/teleop_ps4.yaml" />

Reply:
Oh great, thanks Mike!

Reply:
I believe this change would actually prevent the PS4 controller from launching correctly if JACKAL_PS4 is set to 1, wouldn't it?

Reply:
It's a double negative— "unless JACKAL_PS4 is 0, do the thing"

Reply:
Isn't it "Unless ((JACKAL_PS4 is defined & true) or 0)"?  The 0 is the fall-back value if the variable is not defined?

Reply:
http://wiki.ros.org/roslaunch/XML
> $(optenv ENVIRONMENT_VARIABLE) $(optenv ENVIRONMENT_VARIABLE default_value)
    Substitute the value of an environment variable if it is set. If default_value is provided, it will be used if the environment variable is not set. If default_value is not provided, an empty string will be used. default_value can be multiple words separated by spaces.

Reply:
Oh shoot, so you're right. My bad. :(